src/Main.hs

```diff
@@ -32,22 +35,49 @@ options = Options
       <*> optional ( strOption
           ( long "solc-args"
          <> help "Optional solidity compiler arguments" ))
+      <*> optional ( option auto
+          ( long "epochs"
+         <> help "Optional number of epochs to run coverage guidance" ))
 
 opts :: ParserInfo Options
 opts = info (options <**> helper)
   ( fullDesc
   <> progDesc "Fuzzing/property based testing of EVM code"
   <> header "Echidna - Ethereum fuzz testing framework" )
 
+
 main :: IO ()
 main = do
-  (Options f c s) <- execParser opts
+  (Options f c s n) <- execParser opts
   (v,a,ts) <- loadSolidity f (pack <$> c) (pack <$> s)
-  r        <- newMVar (mempty :: Coverage)
-  let prop t = (PropertyName $ show t
-               , ePropertySeqCoverage r (flip checkETest t) a v 10
-               )
-  _ <- checkParallel . Group (GroupName f) $ map prop ts
-  l <- distinctSize <$> takeMVar r
-  putStrLn $ "Coverage: " ++ show l ++ " unique PCs"
-  return ()
+
+  case n of
+    -- RUN WITHOUT COVERAGE
+    Nothing -> do
+      let prop t = (PropertyName $ show t
+                   , ePropertySeq (flip checkETest t) a v 10
+                   )
+
+      _ <- checkParallel . Group (GroupName f) $ map prop ts
+      return ()
```

Comment:
What is the purpose of the `return ()` at the end of both of these cases? iirc, it was originally there b/c `checkParallel` returns an `IO Bool` but I'm not sure we need it anymore

Reply:
We need the `return ()` in the non-coverage case because `checkParallel` returns `IO Bool`, but the second one is unnecessary.